sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -46,7 +61,8 @@ public final class StorageBlobClient: PipelineClient {
         "https://storage.azure.com/.default"
     ]
 
-    private let manager: TransferManager = URLSessionTransferManager.shared
+    private static let manager: TransferManager = URLSessionTransferManager.shared
```

Comment:
It does, yep... This seemed the quickest way to implement this feature without changing the architecture of the client. But changing that around so that we don't have a single shared TransferManager would then become a breaking API change.

I'm okay with that since we're still in the Preview phase, but if we want to do the work to "de-static" the transfer manager now, we could do that too.

Reply:
Some further / parallel discussion at https://github.com/Azure/azure-sdk-for-ios/pull/256#issuecomment-611833760